src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -259,20 +259,21 @@ foam.CLASS({
     {
       name: 'setScrollPos',
       isFramed: true,
-      code: function() {
-        if ( this.state == this.UNLOADED ) return;
-        if ( this.state != this.LOADED ) { this.setScrollPos(); return; }
+      code: async function() {
         let currI = 0;
         for ( let w = 0 ; w < this.data.wizardlets.length ; w++ ) {
           let wizardlet = this.data.wizardlets[w];
-          if (wizardlet === this.data.currentWizardlet){
+          if ( wizardlet === this.data.currentWizardlet ) {
             currI = Math.max(w - 1, 0);
           }
         }
 
-        var padding = this.childNodes[0].childNodes[0].el().offsetTop;
-        var scrollTop = this.childNodes[0].childNodes[currI].el().offsetTop;
-        this.parentNode.el().scrollTop = scrollTop - padding;
+        var el         = await this.parentNode.el();
```

Comment:
If we move this to the first line, then if we're in the wrong state, we won't waste time with the above loop.